### DISCUSSION TITLE
Docker startup issue

### DISCUSSION THREAD

**Discussion by tommy38hk** [hyness/spring-cloud-config-server#80] [General]
I was unable to run this config server for some reason starting this week. Looks like something had changed and preventing the instance to be running. I even tried different tag versions but still getting the same error. It looks like the startup entrypoint is incorrect.

docker run -it --name=spring-cloud-config-server -p 8888:8888 -v config:/config hyness/spring-cloud-config-server:latest
Unable to find image 'hyness/spring-cloud-config-server:latest' locally
latest: Pulling from hyness/spring-cloud-config-server
Digest: sha256:a0a6f85bf0b2c86908ac647373e11abc50cc9a920d0c398f5cb002fce78b2870
Status: Downloaded newer image for hyness/spring-cloud-config-server:latest
ERROR: failed to launch: determine start command: when there is no default process a command is required

**Comment by hyness** [comment]
Not sure how the image passed CI tests with this error.  My apologies.  Please use a `3.1.0` tag of your choosing until I can resolve the issue

**Reply by tommy38hk** [reply]
Thank you for the speedy respond. Yes, 3.1.0 tag is working.
Just wonder how come the tagged version would change as I thought it would be a snsapshot of a build that would never change

**Reply by hyness** [reply]
You were using the `latest` tag.  All docker images, not just this one, update the `latest` tag anytime there are changes.  If you are looking for a stable version the won't change, you need to use the tags representing the version you wish to use.  You can use `latest` and keep it stable by not pulling it after you get the version you want.

With all that said, the bug has been fixed.  The images are building  and should be done within a few minutes.  `latest` and the rest of the `3.1.1` tags should work now.

**Reply by tommy38hk** [reply]
To be honest, I was using jdk8 tag for a while and all the sudden it did not start. Then I began to use 3.1.1 and latest tags.
